{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "5494b3ad_2e10ae75",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 19,
      "author": {
        "id": 1013536
      },
      "writtenOn": "2024-08-20T11:12:26Z",
      "side": 1,
      "message": "Ideally it would not add the errors again, but I can see how this is tricky and that this solution should anyway be implemented as a last line of defense.",
      "revId": "60371e3b7a23c7554f6f56f0715df009b3b593ff",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}